packages/css/src/components/icon-button/icon-button.scss

```diff
@@ -5,8 +5,12 @@
 
 @mixin reset {
```

Comment:
Anything against a new guideline that all mixins include the name of the element they target?

Reply:
The 'rule' I currently use is:

- use `reset` when there is a single class in the file
- use `reset-*` when there is more than one class in the file.

Using `reset-button` for Icon Button doesn't add anything imo, it's already clear what class the reset is for.

Reply:
Yes, same here. I brought it up because it can be helpful to find all resets everywhere that target e.g. a `<button>` or a `<ul>`. Might have saved you some time in this PR, for instance?

Reply:
Hm, not really, I just searched for '<button' 😁 If we had this convention, I'd probably still do that, just in case.

I think I'm kind of negative to neutral on this change. I don't think it adds much, except another thing you have to keep in mind. I could live with it though.

Reply:
Alright, let’s not change it then.